app.py

```diff
@@ -14,42 +16,42 @@
 from s3repo.view import S3View
 
 
-def load_cfg():
+def load_cfg() -> Dict[str, Any]:
     """Load and parse the config."""
     # Get path to config from env and check if it exists.
     env_cfg_path = os.getenv('RWS_CFG')
     if env_cfg_path is None or not os.path.isfile(env_cfg_path):
         raise RuntimeError('Configuration file does not exist.')
 
     # Parse config.
-    cfg = {}
+    cfg: Dict[str, Any] = {}
     with open(env_cfg_path) as cfg_file:
         cfg = json.load(cfg_file)
 
     return cfg
 
 
-def add_gpg_key(gpg_key):
+def add_gpg_key(gpg_key: bytes) -> str:
     """Adds the given keys to the keyring."""
     # Add keys to the keyring.
     cmd = ['gpg', '--batch', '--import']
-    stdout = None
-    with sp.Popen(cmd, stdin=sp.PIPE, stdout=sp.PIPE, stderr=sp.STDOUT) as proc:
+    with sp.Popen(cmd, stdin=sp.PIPE, stdout=sp.PIPE,
```

Comment:
* Not related with commit.
* Since stdout is used after the "with" code block, I would prefer it to remain the same.

Reply:
rolled back